app/src/main/java/com/xmartlabs/template/service/TaskService.kt

```diff
@@ -0,0 +1,32 @@
+package com.xmartlabs.template.service
+
+import com.xmartlabs.template.R
+import com.xmartlabs.template.model.Balance
+import com.xmartlabs.template.model.Entry
+import com.xmartlabs.template.model.EntryRequest
+import com.xmartlabs.template.model.EntryResponse
+import com.xmartlabs.template.model.User
+import io.reactivex.Single
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+interface TaskService {
+  companion object {
+    const val BALANCE_TASK_PATH = R.string.balance_task_url.toString()
+    const val BASE_TASK_PATH = R.string.base_task_url.toString()
+  }
+
+  @GET(BASE_TASK_PATH)
+  fun getEntries(@Path("user_id/entries") user_id: String): Single<List<Entry>>
+
+  @POST()
+  fun createEntry(@Path("user_id/entries") user_id: String, @Body body: EntryRequest): Single<EntryResponse>
```

Comment:
when we are using `@Path` we put the attribute name like `user_id` and it should be declared with this format `{user_id}` so it would be 

```
@POST("/tasks/1/users/{user_id}/entries")
fun createEntry(@Path("user_id") userId: String, @Body body: EntryRequest): Single<EntryResponse>
```